wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -579,55 +589,57 @@ <h2>Dropdown buttons</h2>
                     </a>
                     <div class="t-dark">
-                        <ul role="menu" class="c-dropdown__menu  u-toggle  u-arrow u-arrow--tl u-background">
-                            <li class="c-dropdown__item ">
-                                <a href="/admin/pages/2/move/" class="u-link is-live ">
-                                    Move
-                                </a>
-                            </li>
-                            <li class="c-dropdown__item ">
-                                <a href="/admin/pages/2/copy/" class="u-link is-live ">
-                                    Copy
-                                </a>
-                            </li>
-                            <li class="c-dropdown__item ">
-                                <a href="/admin/pages/2/delete/" class="u-link is-live ">
-                                    Delete
-                                </a>
-                            </li>
-                            <li class="c-dropdown__item ">
-                                <a href="/admin/pages/2/unpublish/" class="u-link is-live ">
-                                    Unpublish
-                                </a>
-                            </li>
-                            <li>
-                                <hr class="c-dropdown__divider">
-                            </li>
-                            <li class="c-dropdown__item">
-                                <a href="#" class="u-link is-live">
-                                    <span title="Live" class="c-indicator c-dropdown__indicator"></span>
-                                    Live
-                                </a>
-                            </li>
-                            <li class="c-dropdown__item">
-                                <a href="#" class="u-link is-draft">
-                                    <span title="Draft" class="c-indicator c-dropdown__indicator"></span>
-                                    Draft
-                                </a>
-                            </li>
-                            <li class="c-dropdown__item">
-                                <a href="#" class="u-link is-live+draft">
-                                    <span title="Live + Draft" class="c-indicator c-dropdown__indicator"></span>
-                                    Live + Draft
-                                </a>
-                            </li>
-                            <li class="c-dropdown__item">
-                                <a href="#" class="u-link is-absent">
-                                    <span title="Absent" class="c-indicator c-dropdown__indicator"></span>
-                                    Absent
-                                </a>
-                            </li>
-                        </ul>
+                        <nav aria-label="Inline dropdown menu example">
```

Comment:
```suggestion
                        <nav aria-label="Inline dropdown menu example" role="navigation">
```

Reply:
As per previous examples, I won't add the `role` attribute here.